backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -16,14 +16,20 @@ import (
 )
 
 var insertShortLinkRowSQL = fmt.Sprintf(`
-INSERT INTO %s (%s, %s, %s, %s, %s)
-VALUES ($1, $2, $3, $4, $5)`,
+INSERT INTO %s (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
+VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)`,
 	table.ShortLink.TableName,
 	table.ShortLink.ColumnAlias,
 	table.ShortLink.ColumnLongLink,
 	table.ShortLink.ColumnCreatedAt,
 	table.ShortLink.ColumnExpireAt,
 	table.ShortLink.ColumnUpdatedAt,
+	table.ShortLink.ColumnOGTitle,
+	table.ShortLink.ColumnOGDescription,
+	table.ShortLink.ColumnOGImageURL,
+	table.ShortLink.ColumnTwitterTitle,
+	table.ShortLink.ColumnTwitterDescription,
+	table.ShortLink.ColumnTwitterImageURL,
 )
 
 type shortLinkTableRow struct {
```

Comment:
Test struct should be updated to include the new fields. I get the following errors when I run integration test script:
```
app/adapter/sqldb/short_link_integration_test.go:117:6: unknown field 'ogTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:118:6: unknown field 'ogDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:119:6: unknown field 'ogImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:120:6: unknown field 'twitterTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:121:6: unknown field 'twitterDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:122:6: unknown field 'twitterImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:130:6: unknown field 'ogTitle' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:131:6: unknown field 'ogDescription' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:132:6: unknown field 'ogImageURL' in struct literal of type shortLinkTableRow
app/adapter/sqldb/short_link_integration_test.go:133:6: unknown field 'twitterTitle' in struct literal of type shortLinkTableRow
```

Reply:
My bad, it's in #830.